notebooks/official/tensorboard/tensorboard_vertex_pipeline.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>component that were -&gt; component that was | components that were</p><p> </p><p>The <code>create_custom_training_job_op_from_component</code> function converts a given component into a custom training job (<code>CustomTrainingJobOp</code>) in Vertex AI.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>